Assets/00_Content/Scripts/Interactables/Kitchens/Kitchen.cs

```diff
@@ -75,6 +86,9 @@ public class Kitchen : Interactable {
 		private void FinishCooking() {
 			Destroy(tickets.Dequeue());
 
+			if (tickets.Count == 1)
+				cookingCounter.Disable();
```

Comment:
You should probably update the counter here as well, currently it does not decrease when something is finished.

Reply:
im not incrementing the counter, im just setting the value from tickets.Count()

Reply:
nvm , im stupid 🐰